preset: fix import of Audio/Subtitle LanguageList "und"

jstebbins opened the pull request:
"und" used to mean "match any" language.  Now it means "match only
undefined" language.  Forgot to translate "und" to new "any" option on
preset import which breaks presets that have a language list with "und"
defined in it.

Note that this *will* break anyones new presets where they explicitly
wanted "undefined", but this is a much smaller population than those
using "any".

**Test on:**

- [ ] Windows 10+  (via MinGW)
- [ ] macOS 10.13+
- [x] Ubuntu Linux

jstebbins committed:
preset: fix import of Audio/SubTitle LanguageList "und"

"und" used to mean "match any" language.  Now it means "match only
undefined" language.  Forgot to translate "und" to new "any" option on
preset import which breaks presets that have a language list with "und"
defined in it.

Note that this *will* break anyones new presets where they explicitly
wanted "undefined", but this is a much smaller population than those
using "any".

bradleysepos commented:
Could we only target pre-1.3.0 preset versions so anyone updating 1.2.x to 1.3.1 after this lands will have success, and those who updated to 1.3.0 will still have to manually recreate their presets?

jstebbins committed:
preset_template: change "und" to "any" in template

This should have no effect on recent preset definitions since new
presets should have all keys defined.  But very old presets may be
missing some keys and need to be filled in with the template defaults.

jstebbins commented:
> Could we only target pre-1.3.0 preset versions

We could, but that would leave everyone who has already upgraded with having to fix their custom preset language lists.  I think the number of people explicitly wanting to match "undefined" language is going to be a much smaller group.  It was more than a year before anyone even requested the ability to select "undefined" tracks.

bradleysepos commented:
Sounds fine. So anyone who upgraded will get the fix, that makes sense.

galad87 commented:
LGTM
